cmd/cloudshell_open/cloudrun.go

```diff
@@ -40,7 +40,7 @@ func deploy(project, name, image, region string, envs []string) (string, error)
 		"--region", region,
 		"--memory", defaultRunMemory,
 		"--allow-unauthenticated",
-		"--set-env-vars", strings.Join(envs, ","))
+		"--update-env-vars", strings.Join(envs, ","))
```

Comment:
Do we need to change this, since we're omitting the env var for secret?
Does --set-env-vars actually clear all + set specified?

Reply:
Yeah, `--set-env-vars` clears and sets.